packages/protocol/engine/clock/events.go

```diff
@@ -6,31 +6,22 @@ import (
 	"github.com/iotaledger/hive.go/runtime/event"
 )
 
-// Events is a collection of events that can be triggered by the Clock.
+// Events contains a dictionary of events that are triggered by the Clock.
 type Events struct {
-	// AcceptanceTimeUpdated is triggered when a Acceptance Tangle Time advances.
-	AcceptanceTimeUpdated *event.Event1[*TimeUpdateEvent]
+	// AcceptedTimeUpdated is triggered when the accepted time is updated.
+	AcceptedTimeUpdated *event.Event1[time.Time]
```

Comment:
Maybe pass the `RelativeTime` in the events instead of time.Time?

Reply:
I don't think that is necessary as hooking to the event of the clock already implies having access to the entire clock object anyway:

```
clock.myEvent.Hook(func(...) {
    // do sth with clock despite not being in the even param
   clock.RelativeTime()
})
```